manifests/charts/istio-control/istio-discovery/templates/reader-clusterrole.yaml

```diff
@@ -16,7 +17,7 @@ rules:
     resources: ["*"]
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
-    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers", "namespaces", "secrets"]
+    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers", "namespaces"]
```

Comment:
I don't think we can remove this. The secret is used to sync CA and establish the trust between clusters.

Reply:
You are right I did not realised those roles were actually machine roles used in the multi-clusters setup. This makes sense now and what we want to do is not possible with the already built-in roles.

That being said, I do not know if it would be worth to have a cluster role to view istio resources that can be aggregated to the k8s view role. What do you think?

Reply:
I think a role that only allows viewing Istio resources is not enough for the Istio components, which also requires things like service discovery. If we include that, it will be an unused role in the manifest. Maybe we could add it to the samples instead and provide explanations and instructions along with the use cases